tests/test_add_takeoff_and_landing_command.py

```diff
@@ -0,0 +1,36 @@
+"""
+Test process
+"""
+
+import dronekit
+
+from modules import add_takeoff_and_landing_command
+
+
+def test_add_takeoff_and_landing_command():
+    """
+    Tests functionality correctness of add_takeoff_and_landing_command
+    """
+    commands = []
```

Comment:
Add a test for non-empty list. This is where Pytest fixtures would be useful, as well as a generalized function with assertions that is called by the test functions.